tests/test_no_ipynb.py

```diff
@@ -22,14 +22,14 @@ def test_ipynb_diff_with_no_change_single() -> None:
     assert expected_output in result.output
 
 
-def test_ipynb_diff_with_no_change_dir(tmpdir: tmpdir) -> None:
+def test_ipynb_diff_with_no_change_dir(tmp_path: tmp_path) -> None:
```

Comment:
The `tmp_path` value is a `pathlib.Path`, so this should be

```suggestion
def test_ipynb_diff_with_no_change_dir(tmp_path: pathlib.Path) -> None:
```